src/tests/Calculator.test.js

```diff
@@ -0,0 +1,57 @@
+import '@testing-library/jest-dom';
+import userEvent from '@testing-library/user-event';
+import { render, screen, fireEvent } from '@testing-library/react';
+import Calculator from '../components/Calculator';
+import App from '../App';
+
+const labels = ['AC', 'DEL', '%', '/', '*', '-', '+', '=', '.', '0', '1', '2', '3', '4', '5', '6', '7', '8', '9'];
+const numbers = ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9'];
+const ops = ['AC', 'DEL', '%', '/', '*', '-', '+', '=', '.'];
+
+describe('Calculator', () => {
+  test('renders Display component', () => {
+    render(<Calculator />);
+    expect(screen.getByRole('textbox')).toBeInTheDocument();
+  });
```

Comment:
- [ ] Following the project requirements, there should be the usage of Jest snapshots for testing all the components.

Reply:
Bro, I don't quite get it. Is that react-test-renderer?